packages/playwright-core/src/server/fetch.ts

```diff
@@ -422,14 +422,35 @@ export abstract class APIRequestContext extends SdkObject {
 
         const chunks: Buffer[] = [];
         const notifyBodyFinished = () => {
+          function relativeTime(time: number | undefined): number {
+            if (!time)
+              return -1;
+            return time - startAt;
+          }
+          const endAt = monotonicTime();
+          // spec: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
+          const timing: channels.ResourceTiming = {
+            startTime: startAt,
+            domainLookupStart: dnsLookupAt ? 0 : -1,
+            domainLookupEnd: dnsLookupAt ? dnsLookupAt! - startAt : -1,
```

Comment:
`relativeTime()`?

Reply:
done in https://github.com/microsoft/playwright/pull/32647/commits/b1d523b009dc99d856a9dd870dfd639f4671fd00